sdk/storage/azure-storage-blob/azure/storage/blob/_shared/encryption.py

```diff
@@ -187,7 +466,21 @@ def _dict_to_encryption_data(encryption_data_dict):
     else:
         key_wrapping_metadata = None
 
-    encryption_data = _EncryptionData(decode_base64_to_bytes(encryption_data_dict['ContentEncryptionIV']),
+    # AES-CBC only
+    encryption_iv = None
+    if 'ContentEncryptionIV' in encryption_data_dict:
+        encryption_iv = decode_base64_to_bytes(encryption_data_dict['ContentEncryptionIV'])
+
+    # AES-GCM only
+    region_info = None
+    if 'EncryptedRegionInfo' in encryption_data_dict:
+        encrypted_region_info = encryption_data_dict['EncryptedRegionInfo']
+        region_info = _EncryptedRegionInfo(encrypted_region_info['EncryptedRegionDataLength'],
+                                          encrypted_region_info['NonceLength'],
```

Comment:
Super nit: Line 479 and 480 could be moved up 1 whitespace to line up with 478 😛